prod-sample/sample-modules/frank-lloyd-root/0.0.3/src/components/FrankLloydRoot.jsx

```diff
@@ -0,0 +1,93 @@
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import ImmutablePropTypes from 'react-immutable-proptypes';
+import { fromJS } from 'immutable';
+import ModuleRoute from 'holocron-module-route';
+import { holocronModule } from 'holocron';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import HelloWorldComponent from './HelloWorld';
+import Partial, { onPartialRouteEnter } from './Partial';
+
+export const defaultIcon = 'http://localhost:3001/static/modules/frank-lloyd-root/0.0.3/assets/pwa-icon-192px.png';
+export const appleIcon = 'http://localhost:3001/static/modules/frank-lloyd-root/0.0.3/assets/pwa-icon-180px.png';
+// eslint-disable-next-line camelcase
+export const theme_color = '#FDB92D';
+
+export function FrankLloydRoot({ children, config }) {
+  return (
+    <React.Fragment>
+      <Helmet
+        title="Frank Lloyd Root"
+        meta={[
+          { name: 'viewport', content: 'width=device-width, initial-scale=1.0' },
+          { name: 'description', content: 'A Progressive Web App ready Holocron Module' },
+          { name: 'theme-color', content: theme_color },
+        ]}
+        link={[
+          { rel: 'webmanifest', href: `${config.get('cdnUrl')}modules/frank-lloyd-root/0.0.3/assets/manifest.webmanifest` },
+          // pre-connect to Holocron Module CDN
+          { rel: 'preconnect', href: 'http://localhost:3001' },
+          // favicon
+          { rel: 'icon', href: defaultIcon },
+          // icon ideally with a size of 192px (or 180px), it is added as the home icon
+          { rel: 'apple-touch-icon', href: appleIcon },
+        ]}
+      />
+      <pre className="value-provided-from-config">{ config.get('someApiUrl') }</pre>
+      { children }
+    </React.Fragment>
+  );
+}
+
+FrankLloydRoot.childRoutes = (store) => ([
+  <ModuleRoute path="vitruvius" moduleName="vitruvius-franklin" />,
+  <ModuleRoute path="/success" component={HelloWorldComponent} />,
+  <ModuleRoute path="healthy-frank" moduleName="healthy-frank" />,
+  <ModuleRoute path="demo/:moduleName" moduleName="preview-frank" />,
+  <ModuleRoute path="html-partial/:locale/:moduleName" component={Partial} onEnter={onPartialRouteEnter(store)} />,
+]);
+
+FrankLloydRoot.propTypes = {
+  children: PropTypes.node.isRequired,
+  config: ImmutablePropTypes.map.isRequired,
+};
+
+if (!global.BROWSER) {
+  // eslint-disable-next-line global-require
+  FrankLloydRoot.appConfig = require('../config').default;
+}
+
+const reducer = (state = fromJS({})) => state;
+reducer.buildInitialState = ({ req } = {}) => {
+  if (req && req.body) {
+    return fromJS({ postProps: req.body });
+  }
+  return fromJS({});
+};
+
+export default compose(
+  connect((state) => ({ config: state.get('config') })),
+  holocronModule({
```

Comment:
should use the new holocron static rather than the deprecated api - https://github.com/americanexpress/holocron/pull/27/commits/aefb67b72d50ea4dac81b02c7b6c08a6e93ea92e

Reply:
Good call! I've updated dfb5fea